src/lib.rs

```diff
@@ -148,8 +161,8 @@
 //! improve performance of log messages at levels that are disabled. In the
 //! case of our example logger, we'll want to set the maximum log level to
 //! [`Info`][level_link], since we ignore any [`Debug`][level_link] or
-//! [`Trace`][level_link] level log messages. A logging implementation
-//! should provide a function that wraps a call to [`set_logger`], handling
+//! [`Trace`][level_link] level log messages. A logging framework should
+//! provide a function that wraps a call to [`set_logger`], handling
```

Comment:
These changes of "implementation" to "framework" seem indicative of a problem. Previously there was a patch merged that converted "framework" to "implementation", and I assume you did not mean to change these all to "framework". Can you take a look at this diff and see if all these changes are intended? It may be prudent to re-start from the master branch and just make the intended change again.